tests/test_parser.py

```diff
@@ -3,12 +3,10 @@
 
 from ruuvitag_ble import RuuvitagBluetoothDeviceData
 
-OUTDOOR_SENSOR_DATA = (
+V5_SENSOR_DATA = (
```

Comment:
`V5_OUTDOOR_SENSOR_DATA`?

Reply:
I renamed because `OUTDOOR` didn't really bring anything to the party.

Reply:
It makes sense for a human to eyeball-verify that the data parsed from the blob corresponds to an outdoor scene, hence the naming :)

Reply:
I don't necessarily see how the ruuvi data parser would care where the data was from, but either way I went with your suggestion.